### PR TITLE
adds a few new plushies! and also phobias. it's-- it's complicated

### PR DESCRIPTION
**Pull request by Varlaisvea**
yeah okay so
i was adding plushies
and then one of the plushies i wanted to add was a slime plushie.
and it's cute. and i was like "wait, this should set off the slimephobia thing right?"
so i started looking at phobia code. 

and then i remembered there's no mild version of phobias for when you don't want your screen spammed with redtext over the mere presence of a skrell or a dead spider.
so now there is!

### CODE REVIEW

**Comment by Varlaisvea** [comment]
specific plushies this adds:
- bee plushie (also found in hydro bee crates. it's cute.)
- slime plushie (get blennophobed nerd)
- robot plushie (it's animated and everything!)
- shark plushie (it's not explicitly called a blahaj ingame but like...)
- trans shark plushie (...space shark supports u)
- siffet plushie (it's a siffet it's adorable....)

OG arachnophobia and blennophobia are labelled as "severe" now
the "mild" variation isn't afraid of a few of the smaller stuff (dead spiderling goo, mucus, skrell)

and ENTIRELY unrelated but the station perimeter is now capable of being powered if you make an apc there for Making Extensions since the blueprints are black magic spacefuckery that doesn't work on cynosure anyway

**Comment by Cerebulon** [comment]
I think just on the principle of the thing the APC tweak should probably be... Not bundled with a random other thing?
Arguably this is three completely unrelated PRs but at least the slime tweak is related to slimeplushies?

**Comment by Varlaisvea** [comment]
oh yeah it is completely entirely unrelated and is exclusively in this pr because adhd went "oh i should do this while i'm here"
i am a Clown

File: maps/cynosure/cynosure_areas.dm [hunk 1]
```diff
@@ -49,6 +49,7 @@
 // The area near the station, so POIs don't show up right next to the outpost.
 /area/surface/outside/plains/station
 	name = "Station Perimeter"
+	always_unpowered = FALSE //So engineers can make expansions if they provide an APC.
```
Comment (Atermonera) [inline maps/cynosure/cynosure_areas.dm:52]: But you're not "just touching this file", and this isn't just a driveby change
```suggestion
```

File: code/modules/mob/_modifiers/traits_phobias.dm [hunk 2]
```diff
@@ -416,13 +423,19 @@
...
+		if(istype(thing, /obj/item/toy/plushie/slime)) //Squish squish.
 			fear_amount += 1
 
 		if(istype(thing, /obj/effect/decal/cleanable/mucus)) // Blennophobia apparently includes mucus, so!
-			fear_amount += 2
+			if(severe)
+				fear_amount += 2
 
 		if(istype(thing, /obj/item/slime_extract)) // Gooey.
 			fear_amount += 1
```
Comment (Atermonera) [inline code/modules/mob/_modifiers/traits_phobias.dm:441]: Generalize this to a list associating types with fear amounts.
Make the severe versions append to the minor versions' lists of things (Arguably should overwrite b/c the fear amount from the actually-spooky-things should be higher)

File: code/game/objects/items/toys/toys.dm [hunk 3]
```diff
@@ -1219,6 +1219,42 @@
 	icon_state = "fumoplushie"
 	pokephrase = "I just don't think about losing."
 
+//yall out here arguing squids v foxes when blahaj exists???
+/obj/item/toy/plushie/shark
+	name = "shark plushie"
+	desc = "Three feet of condensed love and recycled polyester. It's incredibly huggable."
+	w_class = ITEMSIZE_SMALL //it might be compressible but it's still 3ft long
+	icon_state = "blahaj"
```
Comment (Atermonera) [inline code/game/objects/items/toys/toys.dm:1227]: what

**Review by Atermonera — CHANGES_REQUESTED** [review]

File: code/game/objects/items/toys/toys.dm (hunk 3 above)
Comment (Varlaisvea) [reply code/game/objects/items/toys/toys.dm:1227]: shark plushie!

Comment (Atermonera) [reply code/game/objects/items/toys/toys.dm:1227]: "blahaj"?

Comment (Varlaisvea) [reply code/game/objects/items/toys/toys.dm:1227]: https://en.wikipedia.org/wiki/Bl%C3%A5haj

the internal icon_state name is 'blahaj' because blahaj is a type of large shark plushie made by IKEA which has gained a weirdly large following in the transgender community
due to this being many years in the future however, the word 'blahaj' never appears in anything IC 😔

**Comment by Varlaisvea** [comment]
will reopen this with just the plushies at some point, closing for now bc i must've been high when i thought a phobia refactor would be a good idea in a plushie pr
